src/display/display_utils.js

```diff
@@ -803,7 +803,17 @@ function changeLightness(r, g, b, lumCallback = l => (1 + Math.sqrt(l)) / 2) {
   return `hsl(${h}, ${s}%, ${newL}%)`;
 }
 
+function applyOpacity(r, g, b, opacity) {
```

Comment:
Similar to what we did for `changeLightness` please add a unit test for this function to the `display_utils` tests.